infra/src/Pos/Infra/Communication/Protocol.hs

```diff
@@ -141,14 +141,14 @@ data SpecError
 instance Exception SpecError
 
 instance Buildable SpecError where
-    build (OutSpecNotReported outSpecs spec) =
+    build (OutSpecNotReported outSpecs' spec) =
```

Comment:
Why was that needed? Probably best not to change things that don't *need* to be changed.